### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by plam-ml**

### CODE REVIEW

File: pages/faq.mdx [hunk 1]
```diff
@@ -1,105 +1,80 @@
...
-##### My NFTs are in my wallet, but I can’t see the images, why?
-
-Sui Wallet recognizes images using a standard called [Object Display](https://docs.sui.io/build/sui-object-display). If an NFT doesn’t use this standard, Sui Wallet shows the asset but with no image.
-
-##### How do I send SUI to another person?
+#### How do I get SUI into my wallet?
+Receive SUI from another address, through exchanges or directly within the Sui Wallet using a credit card. Partners like Coinbase, MoonPay, and Transak facilitate these purchases within the wallet.
 
-To send SUI to another wallet, you just need the wallet address. Note you can send only to wallets that support the SUI token, and use 32 or 64 byte addresses. Simply use the **Send** button in your wallet, enter the amount and the recipient wallet address. Ensure to double check the wallet address, you can’t reverse completed transactions. SUI cannot be recovered if you entered an incorrect address.
+#### Who are MoonPay, Coinbase and Transak?
```
Comment (mamos-mysten) [inline pages/faq.mdx:28]: `Banxa` as well

**Review by mamos-mysten — APPROVED** [review]

File: pages/faq.mdx [hunk 2]
```diff
@@ -1,105 +1,80 @@
 # Sui Wallet FAQ
 
-Are you looking for Sui Wallet technical docs? Visit [Sui Wallet by Mysten Labs](./).
+Are you looking for Sui Wallet technical docs? Visit [Sui Wallet by Mysten Labs](https://docs.mystenlabs.com/).
```
Comment (Jordan-Mysten) [inline pages/faq.mdx:3]: Can we just remove this? This doesn't link to technical docs

Comment (plam-ml) [reply pages/faq.mdx:3]: I pinged this q in the joined channel. If hes not responding in time. We can merge and return to change after if otherwise.

Comment (plam-ml) [reply pages/faq.mdx]: removed this copy

**Pull request merged**